### PR TITLE
Add adidas GLITCH to showcase

### PR DESCRIPTION
**Pull request by istvanmakary**
At POSSIBLE we've just launched our React Native application for Adidas! The app is written entirely in RN and available on both iOS and Android.

### CODE REVIEW

**Comment by istvanmakary** [comment]
Hey!

@hramos, @jameshen17 
Do you have any issues with our pull request? If there are please let us know.

Thanks,
Istvan

**Comment by hramos** [comment]
Hey @istvanmakary, the "pinned" list is reserved as noted in the comment. You'll want to add this to the very end of the "featured" list instead, in showcase.js. Please also make sure you meet all the requirements noted in the comments in that file.

File: website/src/react-native/showcase.js [hunk 1]
```diff
@@ -437,14 +437,23 @@ var pinned = [
     defaultLink: 'https://itunes.apple.com/app/apple-store/id1087973225?pt=45076&ct=site-promo&mt=8',
   },
   {
+    name: 'adidas GLITCH',
```
Comment (hramos) [inline website/src/react-native/showcase.js]: See comment in line 334. This is not where you want to add this app.

**Comment by istvanmakary** [comment]
Thanks @hramos, I moved our app to the end of the "featured" list!

**Comment by hramos** [comment]
New location is good, but we still need to make sure the app meets the requirements.

According to the guidelines in that file, "infoLink" must be one of:

 1/ An English-language news article discussing the app, built either by a funded startup or for a public company
2/ An English-language technical post on a funded startup or public company blog discussing React Native

**Comment by istvanmakary** [comment]
I have updated the "infoLink" field!

Thanks for your help @hramos

File: website/src/react-native/showcase.js [hunk 2]
```diff
@@ -444,7 +453,7 @@ var pinned = [
     infoLink: ' https://medium.com/walmartlabs/react-native-at-walmartlabs-cdd140589560#.ueonqqloc',
     infoTitle: 'React Native at Walmart Labs',
     defaultLink: 'https://itunes.apple.com/us/app/walmart-app-shopping-savings/id338137227?mt=8',
-  },
+  }
```
Comment (hramos) [inline website/src/react-native/showcase.js]: Can you add the comma back?

File: website/src/react-native/showcase.js [hunk 3]
```diff
@@ -327,7 +327,16 @@ var featured = [
...
-  },  
+  },
+  {
+    name: 'adidas GLITCH',
+    icon: 'http://d2w4pyf30enukh.cloudfront.net/prod/glitch-logo.jpg/tnm.jpg',
+    linkAppStore: 'https://itunes.apple.com/gb/app/adidas-glitch-interchangeable/id1166181522?mt=8&cm_sp=FOOTBALL-_-UNDEFINED-_-CTA-DOWNLOAD-IPHONE',
+    linkPlayStore: 'https://play.google.com/store/apps/details?id=com.adidas.glitch&cm_sp=FOOTBALL-_-UNDEFINED-_-CTA-DOWNLOAD-ANDROID',
+    infoLink: 'http://www.campaignlive.co.uk/article/adidas-launches-glitch-boot-invite-only-app-revolutionary-approach/1416060',
+    infoTitle: 'GLITCH - Break the pattern',
+    defaultLink: 'http://www.campaignlive.co.uk/article/adidas-launches-glitch-boot-invite-only-app-revolutionary-approach/1416060',
```
Comment (hramos) [inline website/src/react-native/showcase.js]: You don't need a defaultLink here, as this is only used for apps in the pinned array.

File: website/src/react-native/showcase.js [hunk 4]
```diff
@@ -327,7 +327,16 @@ var featured = [
...
     infoTitle: 'React Native in an Existing iOS App: Getting Started'
-  },  
+  },
+  {
+    name: 'adidas GLITCH',
+    icon: 'http://d2w4pyf30enukh.cloudfront.net/prod/glitch-logo.jpg/tnm.jpg',
+    linkAppStore: 'https://itunes.apple.com/gb/app/adidas-glitch-interchangeable/id1166181522?mt=8&cm_sp=FOOTBALL-_-UNDEFINED-_-CTA-DOWNLOAD-IPHONE',
+    linkPlayStore: 'https://play.google.com/store/apps/details?id=com.adidas.glitch&cm_sp=FOOTBALL-_-UNDEFINED-_-CTA-DOWNLOAD-ANDROID',
+    infoLink: 'http://www.campaignlive.co.uk/article/adidas-launches-glitch-boot-invite-only-app-revolutionary-approach/1416060',
+    infoTitle: 'GLITCH - Break the pattern',
```
Comment (hramos) [inline website/src/react-native/showcase.js]: Ideally this title would describe what `infoLink` is about.

**Comment by istvanmakary** [comment]
Sorry, i put it back...

**Comment by mkonicek** [comment]
The link should be about the app (ideally a technical blog post), not a link about the shoe (also has a marketing popup that the user has to close to read the content): http://www.campaignlive.co.uk/article/adidas-launches-glitch-boot-invite-only-app-revolutionary-approach/1416060

**Comment by hramos** [comment]
Closing as the article does not meet our requirements right now. Feel free to reopen if you have a new link here.

**Pull request closed without merging**